Documentation/api-reference.md

```diff
@@ -1,4 +1,4 @@
-## Adobe Experience Platform Media for Edge Network Extension Android API Reference
```

Comment:
can we also add the toc and prerequisites as the other extensions - https://github.com/adobe/aepsdk-edgeidentity-android/blob/main/Documentation/api-reference.md